spec/models/application_record_spec.rb

```diff
@@ -106,10 +106,10 @@
 
   describe 'interview_count' do
     before :each do
-      create :application_record, id: 1
-      create :application_record, id: 2
-      create :interview, application_record_id: 1
-      @collection = ApplicationRecord.where(id: 1)
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
@dfaulken Why not just make `record_1` an instance variance instead? This seems like an unnecessary database query.

Even if it was to be kept, wouldn't it be better to replace it with `.find(record_1.id)` or add `.take` to the end to account for the situation when the query returns more than one result? (abstraction for future addition to test suite since `.where` returns an ActiveRecord relation)